development/idp-local/UserPass.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Example authentication source - username & password.
+ *
+ * This class is an example authentication source which stores all username/passwords in an array,
+ * and authenticates users against this array.
+ *
+ * @author Olav Morken, UNINETT AS.
+ * @package SimpleSAMLphp
+ */
+class sspmod_exampleauth_Auth_Source_UserPass extends sspmod_core_Auth_UserPassBase {
```

Comment:
I had to modify the core library because it stripped out second levels of the data array. So it is just in the local development when using the UserPass auth module that this matters.